### ISSUE TITLE
TypeError when used with node -r flag

### ISSUE THREAD

**Issue by albertogasparin** [testdouble/testdouble.js#226]
When testdouble is used in a file required with the `-r` flag, it breaks with the following error:

```
/td-bug/node_modules/quibble/lib/quibble.js:82
    request = quibble.absolutify(request, parent.filename);
                                                ^

TypeError: Cannot read property 'filename' of null
    at Function.fakeLoad [as _load] (/td-bug/node_modules/quibble/lib/quibble.js:82:49)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:393:7)
    at startup (bootstrap_node.js:150:9)
    at bootstrap_node.js:508:3
```

In order to reproduce this error, create a `mocks.js` file:

```js
const td = require('testdouble');
let myModuleMock = td.func();
td.replace('./my-module.js', myModuleMock);
```

And then try to require it using the `-r` flag (`index.js` can be an empty file):

```sh
node -r ./mocks.js index.js
```

**Comment by searls** [comment]
I tried replicating this but I don't even know what you're trying to accomplish in this usage. Can you explain?

**Comment by searls** [comment]
Landed in 2.1.2
